windows-agent/internal/proservices/landscape/executor_test.go

```diff
@@ -223,6 +291,39 @@ func TestInstall(t *testing.T) {
 	}
 }
 
+//nolint:revive // Context goes after testing.T
+func mockRootfsFileServer(t *testing.T, ctx context.Context) string {
+	t.Helper()
+
+	mux := http.NewServeMux()
+
+	mux.HandleFunc("/goodfile", func(w http.ResponseWriter, r *http.Request) {}) // Return empty file
```

Comment:
I would use the 1.22 syntax: `"GET /goodfile"` (if I’m correct) to restrict to what we support.